src/site/content/en/blog/access-devices/index.md

```diff
@@ -0,0 +1,220 @@
+mobile devices.
+
+If it doesn't work for you, a low-level API may be the solution. Check out
+[Discover how to communicate with your device] to start your journey.
+
+## 🛰 Access GPS coordinates on this device {: #gps }
+
+Use the [Geolocation API] to get the latitude and longitude of the user's
+current position on this device.
+
+If it doesn't work for you, a low-level API may be the solution. Check out
+[Discover how to communicate with your device] to start your journey.
+
+## 🔋 Check the battery on this device {: #batttery }
+
+Use the [Battery API] to get host information about the battery charge level and
+be notified when the battery level or charging status change.
+
+If it doesn't work for you, a low-level API may be the solution. Check out
+[Discover how to communicate with your device] to start your journey.
+
+## 📞 Communicate with this device over the network {: #network }
+
+In the local network, use the [Remote Playback API] to broadcast audio and/or
+video on a remote playback device (e.g. a smart TV or a wireless speaker) or use
+the [Presentation API] to render a web page on a second screen (e.g. a secondary
+display connected with an HDMI cable or a smart TV connected wirelessly).
+
+If this device exposes a web server, use the [Fetch API] and/or [WebSockets] to
+fetch some data from this device by hitting appropriate endpoints. While TCP and
+UDP sockets are not available on the web, check out [WebTransport] to handle
+interactive, bidirectional, and multiplexed network connections. Note that
+[WebRTC] can also be used to communicate data in real-time with other browsers
+using a peer-to-peer protocol.
+
+## 🧱 Discover how to communicate with your device {: #discover }
+
+The decision of what low-level API you should use is determined by the nature of
+your physical connection to the device. If it is wireless, check out Web NFC for
+very short-range wireless connections and Web Bluetooth for nearby wireless
+devices.
+
+- With [Web NFC], read and write to this device when it's in close proximity to
+  the user's device (usually 5–10 cm, 2–4 inches). Tools like [NFC TagInfo by
+  NXP] allow you to browse the content of this device for reverse-engineering
+  purposes.
+
+- With [Web Bluetooth], connect to this device over a Bluetooth Low Energy
+  connection. It should be pretty easy to communicate with when it uses
+  standardized Bluetooth GATT services (such as the battery service) as their
+  behavior is [well-documented]. If not, at this point, you either have to find
+  some hardware documentation for this device or reverse-engineer it. You can
+  use external tools like [nRF Connect for Mobile] and built-in browser tools
+  such as the internal page `about://bluetooth-internals` in Chromium-based
+  browsers for that. Check out [Reverse-Engineering a Bluetooth Lightbulb] from
+  Uri Shaked. Note that HID and serial devices may also be using Bluetooth.
```

Comment:
The intention of this note is to let developers know that they may want to check out the WebHID or Web Serial APIs, even for Bluetooth devices. Maybe reversing this sentence (i.e. "Note that Bluetooth devices may also speak the HID or serial protocols.") would make that clearer.